src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,286 @@
+package duke;
+
+import duke.commands.Command;
+import duke.commands.CommandHandler;
+import duke.tasks.TaskManager;
+import javafx.application.Application;
+import javafx.scene.Scene;
+import javafx.scene.control.Button;
+import javafx.scene.control.Label;
+import javafx.scene.control.ScrollPane;
+import javafx.scene.control.TextField;
+import javafx.scene.layout.AnchorPane;
+import javafx.scene.layout.Region;
+import javafx.scene.layout.VBox;
+import javafx.stage.Stage;
+import java.util.Scanner;
```

Comment:
Should the java imports be at the top? I noticed the same issues in several other places too.